src/v2/components/Validators/Detail/index.jsx

```diff
@@ -216,32 +175,8 @@ const ValidatorsDetail = ({match}: {match: Match}) => {
           )}
           {map(renderSpec)(specs)}
         </ul>
-        <div>
-          <ComposableMap
-            projectionConfig={mapConfig.projection}
-            width={390}
-            height={240}
-            style={mapConfig.style}
-          >
-            <ZoomableGroup center={mapConfig.center} disablePanning>
-              <Geographies
-                geography={`${process.env.PUBLIC_URL}/resources/world-50m-simplified.json`}
```

Comment:
if we're deleting this code, should we delete the `react-simple-maps` dependency & geography resource?

Reply:
It is still being used Bx2PanelValidatorDetail.jsx and Bx2PanelValidatorsOverview.jsx

Reply:
ok cool, I'll trust you to keep it in mind when we remove that old code soon... I think we're getting close *hopefully!*